tests/pipeline/test_blaze.py

```diff
@@ -1377,7 +1393,87 @@ def test_deltas(self, asset_info, add_extra_sid):
                 compute_fn=np.nanmax,
             )
 
-    @with_extra_sid
+    @with_ignore_sid()
+    def test_deltas_before_index_0(self, asset_info, add_extra_sid):
```

Comment:
Should this do something with `add_extra_sid`?

Reply:
done, I add an extra asset to the input which is just ignored so the expected output is the same.